### DISCUSSION TITLE
文字转语音服务异常

### DISCUSSION THREAD

**Discussion by Maxwell1312** [ZuodaoTech/everyone-can-use-english#997] [Q&A]
文字转语音服务时，报错，短句子可以转，长句子转会报错，而且在没有转成功的情况下也扣费了
![Uploading 图片1.png…]()

**Comment by Maxwell1312** [comment]
![image](https://github.com/user-attachments/assets/e415082d-9a03-4f5a-bb6e-f6029b328ea5)

**Reply by an-lee** [reply]
似乎是 cloudflare 有 100 秒的超时限制，超过 100 秒就报超时了。实际上服务器这边可能在第 3 分钟才生成结束。

长文本可以改用 azure/speech 来生成。这个我想想办法看。
